### PR TITLE
fix: use Internet-Draft more consistently across the UI

### LINKED ISSUE #4818: Consistent use of terminology such as Internet-Draft and I-D
### Describe the issue

Datatracker ought to be consistent with established style in using terms like “Internet-Draft” and “I-D”. This is pretty widely used but there are some places where it is not yet. A few examples:

a)  on the [Important Dates page](https://datatracker.ietf.org/meeting/important-dates/):

s/Internet Draft/Internet-Draft
s/for all drafts/for all I-Ds

b) In the Documents menu (screenshot attached):

s/Drafts/I-Ds

c) In the meeting proceedings page (e.g. https://datatracker.ietf.org/meeting/115/proceedings):

s/Drafts/I-Ds
(screenshot attached)

image

image

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by larseggert**
Fixes #4818

### CODE REVIEW

File: ietf/utils/draft.py [hunk 1]
```diff
@@ -630,7 +630,7 @@ def extract_authors(self):
         address_section = r"^ *([0-9]+\.)? *(Author|Editor)('s|s'|s|\(s\)) (Address|Addresses|Information)"
 
         ignore = [
-            "Standards Track", "Current Practice", "Internet Draft", "Working Group",
+            "Standards Track", "Current Practice", "Internet-Draft", "Working Group",
```
Comment (rjsparks) [inline ietf/utils/draft.py:633]: I need to look deeper, but I think this is looking for text produced by another program?

File: ietf/person/models.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ class Person(models.Model):
     biography = models.TextField(blank=True, help_text="Short biography for use on leadership pages. Use plain text or reStructuredText markup.")
     photo = models.ImageField(storage=NoLocationMigrationFileSystemStorage(), upload_to=settings.PHOTOS_DIRNAME, blank=True, default=None)
     photo_thumb = models.ImageField(storage=NoLocationMigrationFileSystemStorage(), upload_to=settings.PHOTOS_DIRNAME, blank=True, default=None)
-    name_from_draft = models.CharField("Full Name (from submission)", null=True, max_length=255, editable=False, help_text="Name as found in a draft submission.")
+    name_from_draft = models.CharField("Full Name (from submission)", null=True, max_length=255, editable=False, help_text="Name as found in an Internet-Draft submission.")
```
Comment (rjsparks) [inline ietf/person/models.py:60]: The changes in this file will require a corresponding migration.

File: ietf/review/models.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ class ReviewerSettings(models.Model):
     min_interval = models.IntegerField(verbose_name="Can review at most", choices=INTERVALS, blank=True, null=True)
     filter_re   = models.CharField(max_length=255, verbose_name="Filter regexp", blank=True,
         validators=[validate_regular_expression_string, ],
-        help_text="Draft names matching this regular expression should not be assigned")
+        help_text="Internet-Draft names matching this regular expression should not be assigned")
```
Comment (rjsparks) [inline ietf/review/models.py:37]: Also needs a migration

**Review by rjsparks — CHANGES_REQUESTED** [review]
There's enough moving here that this deserves multiple reviews. I'm requesting one from Jennifer as well.

File: ietf/name/fixtures/names.json [hunk 4]
```diff
@@ -6025,7 +6025,7 @@
   },
   {
     "fields": {
-      "desc": "The internet drafts ticketing system",
+      "desc": "The Internet-Drafts ticketing system",
```
Comment (jennifer-richards) [inline ietf/name/fixtures/names.json:6028]: These will need to be changed in the production DB, either by migration or manually

**Review by jennifer-richards — COMMENTED** [review]
Did not find anything else alarming, but a couple comments inline.

File: ietf/name/fixtures/names.json (hunk 4 above)
Comment (rjsparks) [reply ietf/name/fixtures/names.json:6028]: Good catch. This is small enough to do by hand using the admin I think.

File: ietf/sync/discrepancies.py [hunk 5]
```diff
@@ -4,25 +4,25 @@
 def find_discrepancies():
     res = []
 
-    title = "Drafts that have been sent to the RFC Editor but do not have an RFC Editor state"
+    title = "Internet-Drafts that have been sent to the RFC Editor but do not have an RFC Editor state"
```
[the remark this answers is not in the document]
Comment (larseggert) [reply ietf/sync/discrepancies.py:7]: Should I undo?

File: ietf/templates/idindex/all_id2.txt [hunk 6]
```diff
@@ -3,29 +3,29 @@
 # generated: {% now "Y-m-d H:i:s T" %}
 #
 # Description of fields:
-# 0   draft name and latest revision
+# 0   Internet-Draft name and latest revision
```
[the remark this answers is not in the document]
Comment (larseggert) [reply ietf/templates/idindex/all_id2.txt:6]: Should I undo?

File: ietf/utils/draft.py (hunk 1 above)
Comment (kesara) [reply ietf/utils/draft.py]: `xml2rfc` always produce `Internet-Draft`.

**Comment by larseggert** [comment]
I think I need some help with the migrations - see CI errors.

File: ietf/sync/discrepancies.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/sync/discrepancies.py:7]: Not yet - still getting input

File: ietf/templates/idindex/all_id2.txt (hunk 6 above)
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/templates/idindex/all_id2.txt:6]: No. I'm happy to get feedback on who uses all_id2.

File: ietf/sync/discrepancies.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (rjsparks) [reply ietf/sync/discrepancies.py:7]: Leave this alone - the RPC confirmed it will not disrupt them.

**Comment by rjsparks** [comment]
Not sure how you built these migrations - what you have at 0031 should have been in the review app and what you have at 0032 is a no-op?
In any case, here's the correction (which I'll push to your branch momentarily):
```
dev ➜ /workspace$ ietf/manage.py showmigrations | grep "\[ "
 [ ] 0031_id_term_2
 [ ] 0030_id_term_1
 [ ] 0032_id_term
dev ➜ /workspace$ ls ietf/person/migrations/003*py
0030_id_term_1.py  0031_id_term_2.py  0032_id_term.py    
dev ➜ /workspace$ rm ietf/person/migrations/003*py
dev ➜ /workspace$ ietf/manage.py makemigrations --name id_term
Migrations for 'person':
  ietf/person/migrations/0030_id_term.py
    - Alter field origin on email
    - Alter field origin on historicalemail
    - Alter field name_from_draft on historicalperson
    - Alter field name_from_draft on person
Migrations for 'review':
  ietf/review/migrations/0031_id_term.py
    - Alter field filter_re on historicalreviewersettings
    - Alter field filter_re on reviewersettings
dev ➜ /workspace$ ietf/manage.py migrate
Operations to perform:
  Apply all migrations: admin, auth, community, contenttypes, dbtemplate, django_celery_beat, doc, group, ipr, liaisons, mailinglists, mailtrigger, meeting, message, name, nomcom, oidc_provider, person, proceedings, review, sessions, sites, stats, submit, tastypie, utils
Running migrations:
  Applying person.0030_id_term... OK
  Applying review.0031_id_term... OK
```
It is nothing but coincidence that these are numbered 0030 and 0031 - it just turns out that's where the counters were in the respective apps.

**Review by rjsparks — APPROVED** [review]

File: ietf/name/fixtures/names.json (hunk 4 above)
Comment (rjsparks) [reply ietf/name/fixtures/names.json:6028]: I made these adjustments in production using the admin

**Pull request merged**